### DISCUSSION TITLE
Pls fix ufc 4

### DISCUSSION THREAD

**Discussion by Jeffy101s** [red-prig/fpPS4#107] [Q&A]
PLS make ufc 4 playable guys it doesn’t boot no signs of life at all I trust you guys pls make it playable

**Comment by ghost** [comment]
Stop this, it isn't like red-prig can magically makes this work within a day or something.

**Reply by Jeffy101s** [reply]
Who asked you?

**Reply by Jeffy101s** [reply]
I just ask can they try and fix it

**Comment by ryzendew** [comment]
Stop wasting peoples time. @red-prig can you close this please

**Comment by red-prig** [comment]
Everything has its time
